src/QueueReceiver.Worker/appsettings.json

```diff
@@ -9,10 +9,14 @@
   "GraphSettings": {
     "Authority": "https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0/",
```

Comment:
I suggest that all these settings are moved to the deployment system and that this file only contains empty fields.